test/transcoding/global_var_decorations.ll

```diff
@@ -0,0 +1,68 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
May I ask you to move this test to a new directory if appropriate?

Reply:
Applied